sdk/resourcemanager/Azure.ResourceManager/src/Resources/Generated/Extensions/ArmResourceExtensions.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using Azure.ResourceManager.Core;
+
+namespace Azure.ResourceManager.Resources
```

Comment:
this isn't needed as an extension and can be placed directly on ArmResource manually